### DISCUSSION TITLE
Missing methods download_ms_buildings  and download_google_buildings

### DISCUSSION THREAD

**Discussion by gtgrp-user** [opengeos/leafmap#687] [Q&A] [answered]
Hi, 
I'm referencing the notebook available at https://leafmap.org/notebooks/81_buildings/, where they used methods like download_ms_buildings and download_google_buildings. However, these methods seem absent in the most recent leafmap version 0.31.2. Is there something evident that I'm overlooking?

**Comment by giswqs** [comment, marked as answer]
It works fine on my local Jupyter and Colab. It is probably your package installation issue. 

https://colab.research.google.com/github/opengeos/leafmap/blob/master/examples/notebooks/81_buildings.ipynb
![image](https://github.com/opengeos/leafmap/assets/5016453/844fff5f-012f-4c78-bf0d-c714de2c6694)

**Comment by gtgrp-user** [comment]
This is indeed a package installation issue, much apreciated thanks.
